### ISSUE TITLE
when url return only one line without \n it display no content

### ISSUE THREAD

**Issue by IBM-lily** [mozilla-frontend-infra/react-lazylog#50]
when use this component with url return a string without line break \n, it display 'No content', please help to check it, thanks very much!

**Comment by helfi92** [comment]
Do you have a `url` I can use to reproduce this issue?

**Comment by IBM-lily** [comment]
sorry about it, I just use local API to get the data from database.
This filed is 'text' in data table like below:
"stderr": "status code: 403, reason phrase: Forbidden"

<LazyLog  url={urlErr}  fetchOptions={options} follow={true}/>
With API (urlErr) to get this filed, it return 'No Content'

update log set stderr=E'status code: 403, reason phrase: Forbidden \n' 
then it returns the data correctly.

Hope it helps to reproduce it, thank you very much!!

**Comment by kunyan** [comment]
I found same issue, you could using this url to reproduce the issue

https://gist.githubusercontent.com/kunyan/69a9cef8b6922ac32480496dcc7f9a40/raw/2941f5365bab6ba8a84e061742c700f89c44d13a/data.log

It will display like

<img width="1126" alt="Screen Shot 2019-11-20 at 10 29 28 AM" src="https://user-images.githubusercontent.com/752130/69203957-aa5a8580-0b80-11ea-8e32-21415442fdf6.png">

**Comment by kunyan** [comment]
I think this part should be rewrite
https://github.com/mozilla-frontend-infra/react-lazylog/blob/master/src/components/LazyLog/index.jsx#L634,L644

for some few lines output. the height will incorrect

The `calculateListHeight` could be 

```
  calculateListHeight = autoSizerHeight => {
    const { height, enableSearch, rowHeight } = this.props;
    const { lines } = this.state;

    const linesHeight = lines * rowHeight;
    const autoHeight = autoSizerHeight > linesHeight ? autoSizerHeight : linesHeight;

    if (enableSearch) {
      return height === 'auto'
        ? autoHeight - SEARCH_BAR_HEIGHT
        : height - SEARCH_BAR_HEIGHT;
    }

    return height === 'auto' ? autoHeight : height;
  };

```

**Comment by helfi92** [comment]
Fixed in react-lazylog@4.4.2 🎉
